scripts/variantstore/wdl/extract/generate_ordered_paths.py

```diff
@@ -0,0 +1,17 @@
+import argparse
+
+def generate_ordered_paths(root_path, path_suffix, number):
+    for i in range(0, number):
```

Comment:
isn't range end-exclusive?  should this be number+1 ? (please verify!)

Reply:
according to https://www.w3schools.com/python/ref_func_range.asp and my test run, it stops before the specified number